src/scenes/ObservationsRouter/BiomassMeasurementsDetails.tsx

```diff
@@ -0,0 +1,367 @@
+      {observation?.adHocPlot?.monitoringPlotName}
+    </Typography>
+  );
+
+  const crumbs: Crumb[] = useMemo(() => {
+    const data: Crumb[] = [];
+
+    if (plantingSiteId) {
+      data.push({
+        name: strings.OBSERVATIONS,
+        to: APP_PATHS.OBSERVATIONS_SITE.replace(':plantingSiteId', plantingSiteId?.toString()),
+      });
+      data.push({
+        name: strings.BIOMASS_MEASUREMENTS,
+        to: `${APP_PATHS.OBSERVATIONS_SITE.replace(':plantingSiteId', plantingSiteId?.toString())}&tab=biomassMeasurements`,
+      });
+    }
+
+    return data;
+  }, [activeLocale, plantingSiteId, observationId]);
+
+  return (
+    <Page crumbs={crumbs} title={title} titleContainerStyle={{ paddingTop: 3, paddingBottom: 1 }}>
+      <Grid container>
+        <Grid item xs={12}>
+          <Card flushMobile>
+            <Typography fontSize='20px' lineHeight='28px' fontWeight={600} color={theme.palette.TwClrTxt}>
+              {strings.DETAILS}
+            </Typography>
+            <Grid container>
+              {data.map((datum, index) => (
+                <Grid key={index} item xs={gridSize} marginTop={2}>
+                  <Textfield
+                    id={`plot-observation-${index}`}
+                    label={datum.label}
+                    value={datum.value}
+                    type={datum.text ? 'textarea' : 'text'}
+                    preserveNewlines={true}
+                    display={true}
+                  />
+                </Grid>
+              ))}
+              <Grid container spacing={2} marginTop={2}>
+                <Grid item xs={6}>
+                  <Grid item xs={12}>
+                    <Typography fontSize='20px' lineHeight='28px' fontWeight={600} color={theme.palette.TwClrTxt}>
+                      {strings.PLOT_PHOTO}
+                    </Typography>
+                  </Grid>
+                  <Grid item xs={12} display={'flex'}>
+                    <Grid item xs={6}>
+                      <MonitoringPlotPhotos
+                        observationId={Number(observationId)}
+                        monitoringPlotId={Number(monitoringPlot?.monitoringPlotId)}
+                        photos={monitoringPlot?.photos.filter(
+                          (photo) => photo.type === 'Plot' && photo.position === undefined
+                        )}
+                      />
+                    </Grid>
+                    <Grid item xs={6}>
+                      <Typography
+                        fontSize='14px'
+                        lineHeight='28px'
+                        fontWeight={400}
+                        color={theme.palette.TwClrTxtSecondary}
+                      >
+                        {strings.DESCRIPTION_NOTES}
+                      </Typography>
+                      <Typography fontSize='16px' lineHeight='24px' fontWeight={500} color={theme.palette.TwClrTxt}>
+                        test
+                      </Typography>
```

Comment:
Should "test" be replaced or removed?

Reply:
It should be replaced with a value from the api. I will update it when I know where to get that info

Reply:
Makes sense, thanks!